recipe/meta.yaml

```diff
@@ -47,12 +46,15 @@ test:
     - nuitka.utils
   requires:
     - pip
+    - {{ compiler('c') }}  # [linux]
```

Comment:
After adding this, you then removed the compilation test `nuitka Empty.py`, so is this still required?

Reply:
I returned the example